src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Models/CommonModels/Enums.cs

```diff
@@ -23,6 +23,8 @@ namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models
     public enum ContainerType
     {
         AzureVM,
+
+        AzureSqlContainer,
```

Comment:
Nit: Do we need "Container" here? Unless we are dependent on service for this enum, can we remove the Container?